frontend/components/atoms/Spelling-bee/Progress.jsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import * as React from "react";
+import * as ProgressPrimitive from "@radix-ui/react-progress";
+
+import { cn } from "@/lib/utils";
+
+const Progress = React.forwardRef((props, ref) => {
```

Comment:
Since React 19, we don't need forwardRef anymore, you can simply pass it as props/

https://react.dev/reference/react/forwardRef